pages/home/HomePage.jsx

```diff
@@ -36,26 +33,18 @@ var HomePage = React.createClass({
     fixLocation(this.context.location);
   },
   componentDidMount: function() {
-    // auto pops up the Pledge modal if the user is visiting
-    // the homepage for the first time
-    var disableModal = "disableAutoPledgeModal";
-    if (!localStorage[disableModal]) {
-      this.handlePledgeBtnClick();
-      localStorage.setItem(disableModal, "disabled");
-    }
-
-    if (this.context.location.search.pledge === "thanks") {
+    if (this.context.location.search.signup === "thanks") {
       this.props.showModal(ThankYouModal, {
         hideModal: this.props.hideModal
       });
       // Optimizely conversion tracking
       window.optimizely = window.optimizely || [];
-      window.optimizely.push(['trackEvent', 'pledgeFormSubmitted']);
+      window.optimizely.push(['trackEvent', 'NewsletterFormSubmitted']);
     }
   },
-  handlePledgeBtnClick: function() {
-    ga.event({ category: 'Clicked Home CTA', action: 'Pledge to Teach' });
-    this.props.showModal(ModalPledge, {
+  handleEmailBtnClick: function() {
+    ga.event({ category: 'Clicked Home CTA', action: 'Get email updates' });
```

Comment:
I'd rather change the actual string, as we do string testing in the `test.homepage.jsx` file